recipe/meta.yaml

```diff
@@ -21,26 +21,32 @@ source:
     - patches/0002-link-libomp-to-compiler-rt-on-osx-arm.patch
 
 build:
-  number: 0
+  number: 1
 
 outputs:
   - name: llvm-openmp
     script: build-pkg.sh   # [unix]
     script: build-pkg.bat  # [win]
     build:
+      # due to https://github.com/conda/conda-build/issues/5403
+      overlinking_ignore_patterns:  # [ppc64le]
+        - "**"                      # [ppc64le]
       ignore_run_exports:
-        - libgcc-ng
-        - libstdcxx-ng
         - _openmp_mutex
+      ignore_run_exports_from:
+        - {{ compiler("cxx") }}
+        - {{ compiler("fortran") }}
```

Comment:
The `.mod` files are specific to the gfortran version (maybe not the major version, but the soversion. not sure), so it's not possible to remove these.